src/login/translation/messageIndex.js

```diff
@@ -40,12 +40,36 @@ export const unformattedMessages = defineMessages({
     defaultMessage: `Email confirmation code`,
     description: "Placeholder for email text input"
   },
+  "emails.invalid_code": {
+    id: "emails.invalid_code",
+    defaultMessage: `Invalid confirmation code`,
+    description: "error text for invalid confirmation code"
+  },
+  "emails.confirm_code_wrong_length": {
+    id: "emails.confirm_code_wrong_length",
+    defaultMessage: `Email confirmation code must be 36 characters long`,
+    description: "Error message for wrong length",
+  },
+  "mobile.confirm_code_wrong_length": {
+    id: "mobile.confirm_code_wrong_length",
+    defaultMessage: `Phone confirmation code must consists of 10-12 characters`,
```

Comment:
extra S here: "must consist of" (but for consistency I'd consider aligning this with the 36 char message, i.e. "code must be XX characters long") and is it any character (or only letters and numbers)?

Reply:
English: Code must be 10-12 characters long, only letters and numbers allowed
Swedish: Koden måste vara 10-12 tecken lång, endast bokstäver och nummer tillåtna